documentation/getting_started/getting_started_01_basic_setup.md

```diff
@@ -157,12 +157,13 @@ defmodule MyApp.Accounts.User do
     api MyApp.Accounts
 
     strategies do
-      password :identity_field :email
+      password :password, identity_field: :email
```

Comment:
This is what happens when I'm typing furiously trying to get something done by the end of the day!

```suggestion
      password :password do
        identity_field :email
      end
```